src/webserver/HttpBase.h

```diff
@@ -437,11 +438,10 @@ handle_request(
                     RPC::makeError(RPC::RippledError::rpcBAD_SYNTAX))));
 
         boost::json::object response;
-        auto start = std::chrono::system_clock::now();
-        auto v = RPC::buildResponse(*context);
-        auto end = std::chrono::system_clock::now();
-        auto us =
-            std::chrono::duration_cast<std::chrono::microseconds>(end - start);
+        auto [v, timeDiff] =
+            util::timed([&]() { return RPC::buildResponse(*context); });
+
+        auto us = std::chrono::duration<int, std::milli>(timeDiff);
```

Comment:
Looks like it changed from microseconds to milliseconds here?

Reply:
I think you can also just do 
```cpp
auto [v, us] = util::timed<std::micro>([&]() { ... });
```

Reply:
:(. i could not . The us has to be duration. the output of timed is absolute duration count in a certain unit.
I think the milli is okay here, the timeDiff is milli, so duration should be milli. The logDuration will cast it to other unit.